activities/organization/finance-updates/finance-update-december-2020.md

```diff
@@ -0,0 +1,47 @@
+---
+type: resource
+---
+
+# Finance update December 2020
+
+## Summary
+
+* Current cash at bank position: € 1.370.316,45
```

Comment:
In American and British English, it's correct to represent this number as €1,370,316.45. Would it be difficult to reverse all commas and decimals from the Dutch financial tracking tools for these reports? It would bring the reports in line with the English used on the rest of our website.

Reply:
It's correct in the financial tools. These one-pagers are made manually, and thus the error is on my side. Thank you for the diligence <3